jobwatch/jobwatch.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, task, filename,
         self.maxage = maxage
         self.filetime = None
         self.filedate = None
-
+        self.type = 'Job'
```

Comment:
This hack was to make a test pass, but I think I should probably just update the code in reporting that looks for a  "jw.type" and skip if type is missing.

Reply:
Does it work if you assign `None`. This would be a bit more agnostic.

Reply:
No, None didn't work.  And setting it to 'Job' seemed to just override the rest of the types.  I didn't have the brainpower just now to figure out the inheritance and just updated the logic in the html report method for now.